samples/wifi_provisioning_over_ble/src/main.c

```diff
@@ -226,9 +228,14 @@ static void handle_in_commands(uint8_t *value, uint16_t len)
 	} break;
 	/*Sending Security type*/
 	case '5': {
```

Comment:
I've only know noticed that there's quite a lot more "magic value" usage in this app. Would be good to have properly named defines for all of these, but definitely out of scope for this PR, so if you feel like it please consider improving in a follow-up PR.